### PR TITLE
UI polishing

### PR DESCRIPTION
**Pull request by matejglejtek**
Completion of UI according to figma design, adjustments to aircraft detail view

### CODE REVIEW

**Review by marianhlavac — CHANGES_REQUESTED** [review]
The project is missing the `assets/config/uasid_prefix.json` file, cannot make the build without the file, app crashes on start. Pls include in the project if it's not sensitive.

File: lib/utils/utils.dart [hunk 1]
```diff
@@ -158,3 +158,19 @@ Image? getFlag(String countryCode) {
   }
   return flag;
 }
+
+Image? getManufacturerLogo({String? manufacturer, Color color = Colors.black}) {
+  if (manufacturer == null) return null;
+  String? path;
+  if (manufacturer == 'Dronetag') path = 'assets/images/dronetag.png';
+  if (manufacturer == 'DJI') path = 'assets/images/dji_logo.png';
```
Comment (marianhlavac) [inline lib/utils/utils.dart:166]: I'd later convert these logos into an icon pack (same as the DroneScannerIcons font) and use the vector variants. Just an improvement, no need to change it in this PR.

**Comment by matejglejtek** [comment]
I swapped icons in list card for ones from design, changed spacing in card and slider headers, adjusted some colors, made flags circular etc. I will test it on different screensizes asap.

**Comment by marianhlavac** [comment]
Screenshot_20220826-095626

There is overflow on the details and the objects on the top of the detail panel are misaligned. The icons inside the buttons are also misaligned (too large and not centered)

**Review by marianhlavac — CHANGES_REQUESTED** [review]
The UI has been improved by this PR greatly, a lot of work went into this, however there are still a few important details missing, overflows are still present and some elements seem misaligned. I'd advise one final check against the Figma design.

Also a few conflicts has appeared, let's resolve then before we can merge.

**Comment by matejglejtek** [comment]
Overflows were fixed, as well as position of side toolbar.

**Review by marianhlavac — CHANGES_REQUESTED** [review]
There are still issues to be fixed, I've shared the list of issues with you.

**Comment by matejglejtek** [comment]
I disabled landscape mode, changed layout of about dialog and showcase items. I corrected grammar is showcase texts. There is also new wifi icon without watermark.
In opendroneid library I fixed a problem that probably caused problem with starting scans.

**Comment by matejglejtek** [comment]
I changed showcase items so they have the same styling, size and are centered to the middle of the screen.
I fixed colors of back buttons in license page and searching for places page.

**Review by marianhlavac — APPROVED** [review]
Alright, looks solid now, good job! Any further UI issues we'll cover in another issues.

**Pull request merged**